docs/src/views/FocusZone.tsx

```diff
@@ -0,0 +1,366 @@
+      <li>
+        <b>disabled</b> - If set, the FocusZone will not be tabbable and keyboard navigation will be
+        disabled. This does not affect disabled attribute of any child.
+        <p>Default: {code('false')}</p>
+        <p>Type: {code('boolean')}</p>
+      </li>
+      <li>
+        <b>isRtl</b> - If true, FocusZone behavior will change to match RTL environments (left/right
+        arrows switched).
+        <p>Default: {code('false')}</p>
+        <p>Type: {code('boolean')}</p>
+      </li>
+      <li>
+        <b>isCircularNavigation</b> - If true, will cycle to the beginning of the targets once the
+        user navigates to the next target while at the end, and to the end when navigate to the
+        previous at the beginning.
+        <p>Default: {code('false')}</p>
+        <p>Type: {code('boolean')}</p>
+        <p>For example, {link('horizontal menu', '/components/menu#types-menu')}.</p>
+      </li>
+      <li>
+        <b>shouldEnterInnerZone</b> - callback function, will be executed on keypresses to determine
+        if the user intends to navigate into the inner (nested) zone. Returning true will ask the
+        first inner zone to set focus. For example, when chat container is focus zone and chat
+        messages are inner focus zones. Navigation between messages possible with up/down arrow
+        keys, but when pressing Enter, focus should go to focusable elements inside message, for
+        example, a link.
+        <p>Default: {code('false')}</p>
+        <p>Type: {code('(ev: React.KeyboardEvent<HTMLElement>) => boolean')}</p>
+        <CodeSnippet
+          label="jsx"
+          value={`
+            <Chat>--->FocusZone          
+             <ChatMessage /> ---> inner FocusZone     
+             <ChatMessage /> ---> inner FocusZone        
+             <ChatMessage /> ---> inner FocusZone  
+            </Chat>
+          `}
+        />
+        <CodeSnippet
+          label="chatBehavior.ts"
+          value={`
+          const chatBehavior: Accessibility = (props: any) => ({
+            //...
+            focusZone: {
+              //...
+              props: {
+                shouldEnterInnerZone: event => keyboardKey.getCode(event) === keyboardKey.Enter,
+              },
+            },
+          })
+          `}
+        />
+      </li>
+      <li>
+        <b>onActiveElementChanged</b> - Callback for when one of immediate children elements gets
+        active by getting focused or by having one of its respective children elements focused.
+        <p>Type: {code('(element?: HTMLElement, ev?: React.FocusEvent<HTMLElement>) => void')}</p>
+      </li>
+      <li>
+        <b>shouldReceiveFocus</b> - Callback method for determining if focus should indeed be set on
+        the given element.
+        <p>Type: {code('(childElement?: HTMLElement) => boolean')}</p>
+      </li>
+      <li>
+        <b>handleTabKey</b> - Allows tab key to be handled to tab through a list of items in the
```

Comment:
Allows tab key to be handled to tab ...

not sure what you meant here. Can it be rephrased?